### ISSUE TITLE
count of vae embedding is less than number of images

### ISSUE THREAD

**Issue by LianShuaiLong** [bghira/SimpleTuner#689]
here is my trainning dataset and correspond count:
<img width="1490" alt="企业微信截图_eb2d24ce-ae26-47ee-8593-797a14dc2864" src="https://github.com/user-attachments/assets/7e2c728e-8957-4931-8576-f729f5b210ae">
<img width="868" alt="企业微信截图_6369bd47-f115-4732-8e6d-a20394578fad" src="https://github.com/user-attachments/assets/4d59195e-4365-437f-bda9-4c267e2c04c4">
and the cache of vae result and corresponding count:
<img width="1414" alt="企业微信截图_9c0f54d5-54ca-48d3-8ae5-0105e452c2f1" src="https://github.com/user-attachments/assets/27b37b2a-4007-422e-9dcb-4b0516cf3194">
<img width="836" alt="企业微信截图_43d77839-cf39-4652-a665-b408d0685ffa" src="https://github.com/user-attachments/assets/2fc729b5-d9d9-4b70-9891-55165d93a18e">

this is my traning configuration:
<img width="763" alt="image" src="https://github.com/user-attachments/assets/8eb7e6a2-641c-4516-b848-5c986f7599fc">

**Comment by bghira** [comment]
yeah.. it has to be a multiple of batch size as stated in the docs

**Comment by LianShuaiLong** [comment]
> yeah.. it has to be a multiple of batch size as stated in the docs

but the num of text embedding in the cache folder is 76,more than the number of image in the train dataset and not equal to the number of image embedding in the vae cache folder; I am totally confused ....
<img width="1359" alt="image" src="https://github.com/user-attachments/assets/a1c100e0-3302-408d-a99a-897a608eced8">

**Comment by bghira** [comment]
possibly your text files have newlines which makes it into multiple captions

**Comment by LianShuaiLong** [comment]
> possibly your text files have newlines which makes it into multiple captions

ok,i will check that;
Another silly question, the text embedding filename seems like hash value, how can I find the corresponding image file through the filename of the text embedding, or how can I keep the filename of the text embedding consistent with the caption file ( like image_1.txt)?

**Comment by bghira** [comment]
it's not an option to do that anymore due to issues with filenames, eg. being too long and it's just `hash()`

enable the `SIMPLETUNER_LOG_LEVEL=DEBUG` and check `debug.log`

**Comment by LianShuaiLong** [comment]
> it's not an option to do that anymore due to issues with filenames, eg. being too long and it's just `hash()` of the actual caption itself. so you probably won't be able to link it backwards.
> 
> enable the `SIMPLETUNER_LOG_LEVEL=DEBUG` and check `debug.log`

thanks
